contrib/k8s/charts/open-service-broker-azure/README.md

```diff
@@ -57,11 +57,12 @@ $ export AZURE_CLIENT_SECRET=<password>
 ## Installing the Chart
 
 Installation of this chart is simple. First, ensure that you've [added the
-`azure` repository](../README.md#installing-charts). Then, install from the
+`azure` repository]. Then, install from the
 `azure` repo:
 
 ```console
-$ helm install azure/open-service-broker-azure --name osba --namespace osba \
+ helm repo add azure https://kubernetescharts.blob.core.windows.net/azure
+ helm install azure/open-service-broker-azure --name osba --namespace osba \
```

Comment:
For consistency, use $ at the start of the lines to indicate a shell prompt.

I do realize that this choice might be a little controversial because of how it affects how markdown highlights things. If we want to change that, we should change it across the board in a separate PR, so here and now, let's just opt for consistency.

Reply:
Uh... I should have said "consistency within _this_ page." Because it looks like overall, we're not consistent throughout the docs.